src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    int MIN_QUANTITY = 0;
```

Comment:
Constants should be declared in a dedicated class or within the classes that use them. It's not a good practice to declare them within an interface as they are implicitly public static final, and can lead to unwanted coupling if used across different classes.